.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,49 @@
+## Contributing guidelines
+
+Thank you for your interest in contributing to EvalAI! Here are a few pointers about how you can contribute.
+
+### Setting things up
+
+To set up the development environment, follow the instructions in [README](https://github.com/Cloud-CV/EvalAI-ngx/blob/master/README.md).
+
+### Finding something to work on
+
+The issue tracker of EvalAI a good place to start. If you find something that interests you, comment on the thread and we’ll help get you started.
+
+Alternatively, if you come across a new bug, please file a new issue and comment if you would like to be assigned. The existing issues are tagged with one or more labels, based on the part of the website it touches, its importance etc., that can help you in selecting one.
+
+If neither of these seem appealing, please post on our [gitter channel](https://gitter.im/Cloud-CV/EvalAI) and we will help find you something else to work on.
+
+### Instructions to submit code
+
+Before you submit code, please talk to us via the issue tracker so we know you are working on it and also let us know your approach, to be sure that you are thinking on right direction.
+
+Our central development branch is development. Coding is done on feature branches based off of development and merged into it once stable and reviewed. To submit code, follow these steps:
+
+1. Create a new branch off of development. Select a descriptive branch name. The commands below will help you to get the latest version of the code from the upstream(master branch).
```

Comment:
How to set the remote `upstream` ?

Reply:
You can add a note, something like:

If you have not added the upstream then run this command:

```
git remote add upstream https://github.com/Cloud-CV/EvalAI-ngx
```

and to verify : 

```
git remote -v
```